roles/ansible-runner/tasks/main.yml

```diff
@@ -27,20 +27,27 @@
   file:
     path: "{{ ansible_runner_virtualenv }}/plugins/callback"
     state: directory
-    owner: "{{ ansible_runner_user }}"
+    mode: 0755
+    owner: root
+    group: root
 
 - block:
   - name: Install datadog callback plugin
     get_url:
       url: "{{ datadog_callback_url }}"
       dest: "{{ ansible_runner_virtualenv }}/plugins/callback/datadog_callback.py"
-      owner: "{{ ansible_runner_user }}"
+      mode: 0755
+      owner: root
+      group: root
 
   - name: Install datadog callback configuration
     copy:
       dest: "{{ ansible_runner_virtualenv }}/plugins/callback/datadog_callback.yml"
       content: "api_key: {{ secrets.datadog.api_key }}"
-      owner: "{{ ansible_runner_user }}"
+      mode: 0755
```

Comment:
@rattboi: Thanks. I made a mental note to set it to 0644, but never actually made the change.